src/build.py

```diff
@@ -1258,6 +1264,9 @@ def Summary(repos):
   print 'Failed steps: %s.' % buildbot.Failed()
   for step in buildbot.FailedList():
     print '    %s' % step
+  print 'Warned steps: %s.' % buildbot.Warned()
+  for step in buildbot.WarnedList():
+    print '    %s' % step
 
   if IsBuildbot():
     latest_file = '%s/%s' % (BUILDBOT_BUILDERNAME, 'latest.json')
```

Comment:
Yeah, it kind of really boils down to who cares about what. The test-dependency thing is solvable the same way regardless. I think this situation came down to, "we care less about JSC and also about LKGR" (i.e. we don't really use it for anything right now) and "JF cares more about JSC and also more about LKGR" (presumably he wouldn't have asked for that behavior if he didn't care). My opinion for now is that if nobody cares enough about JSC to keep it up to date (and this seems empirically true at least right now), then I don't think it should turn the bot red. But I also don't care about LKGR all that much right now either, so I'm happy to defer to what JF wants on that.